CHANGELOG.rst

```diff
@@ -1,6 +1,38 @@
 Changelog
 =========
 
+0.8.1 (2018-12-20)
+------------------
+
+MLflow 0.8.1 introduces several improvements:
+
+- Better UI performance, especially when navigating within experiments containing many runs, parameters, and metrics
+- Improved visualizations, including interactive scatter plots for MLflow run comparisons
+- Expanded support for scoring Python models as Spark UDFs
+
+Features:
+
+- [API/CLI] Support for running MLflow projects from ZIP files (#759, @jmorefieldexpe)
+- [Python API] Support for passing model conda environments as dictionaries to ``save_model`` and ``log_model`` functions (#748, @dbczumar)
+- [Models] Default Anaconda environments have been added to many of MLflow's Python model flavors. By default, models with these flavors that are produced by MLflow's various `save_model` and `log_model` functions will include an environment specifying all of the versioned dependencies that are necessary to load and serve the model (#705, #707, #708, #749, @dbczumar)
```

Comment:
I'd say this is a major improvement -- something like "By default, saved models will include a conda environment capable of loading them back. Previously, users had to pass this environment through manually."

Reply:
Default Anaconda environments have been added to many Python model flavors. By default, models produced by `save_model` and `log_model` functions include a conda environment that specifies all of the versioned dependencies necessary to load and serve the models.  Previously, you had to pass this environment through manually.

Reply:
@aarondav @stbof I've added this feature to the list of *significant* improvements and updated the phrasing to reflect your suggestions